unturned/OpenMod.Unturned/Animals/Events/UnturnedAnimalAddedEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace OpenMod.Unturned.Animals.Events
+{
+    public class UnturnedAnimalAddedEvent : UnturnedAnimalSpawnedEvent
```

Comment:
Whats the difference to AnimalSpawnedEvent?

Reply:
There are two different ways animals can be spawned in Unturned - a new instance being added (UnturnedAnimalAddedEvent) or a dead instance being revived (UnturnedAnimalRevivedEvent).

As these are two different ways animals may be spawned, they are their own separate classes.

If a developer just wants to know when an animal spawns, regardless of how, they can subscribe to the class they both inherit - UnturnedAnimalSpawnedEvent.